Flaky TestExecQueue_Stop

```
--- FAIL: TestExecQueue_Stop (0.00s)
    exec_queue_test.go:54: 
        	Error Trace:	/home/runner/work/ssv/ssv/utils/tasks/exec_queue_test.go:54
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 0
        	Test:       	TestExecQueue_Stop
2025-02-03T15:04:32.812089Z	DEBUG	logger is ready	{"level": "debug", "encoder": "capital", "format": "console", "file_options": null}
2025-02-03T15:04:32.824069Z	DEBUG	logger is ready	{"level": "debug", "encoder": "capital", "format": "console", "file_options": null}
2025-02-03T15:04:32.833731Z	DEBUG	logger is ready	{"level": "debug", "encoder": "capital", "format": "console", "file_options": null}
FAIL
```

https://github.com/ssvlabs/ssv/actions/runs/13116254524/job/36591183364

https://github.com/ssvlabs/ssv/blob/8ff4a74877996718c361ee91ad20f45f79470c92/utils/tasks/exec_queue_test.go#L54

--- a/utils/tasks/exec_queue_test.go
+++ b/utils/tasks/exec_queue_test.go
@@ -42,8 +42,6 @@ func TestExecQueue_Stop(t *testing.T) {
 	var i int64
 	q := NewExecutionQueue(1 * time.Millisecond)
 
-	go q.Start()
-
 	var wg sync.WaitGroup
 	wg.Add(1)
 	q.Queue(func() error {
@@ -52,6 +50,8 @@ func TestExecQueue_Stop(t *testing.T) {
 		return nil
 	})
 	require.Equal(t, 1, len(q.(*executionQueue).getWaiting()))
+
+	go q.Start()
 	wg.Wait()
 	require.Equal(t, 0, len(q.(*executionQueue).getWaiting()))
 

